Maestro/subscriptions.json

```diff
@@ -854,6 +854,8 @@
         "https://github.com/dotnet/sdk/blob/rel/**/*",
         "https://github.com/dotnet/sdk/blob/release/**/*",
         "https://github.com/dotnet/dotnet-cli-archiver/blob/master/**/*",
+        "https://github.com/dotnet/cli-migrate/blob/master/**/*",
+        "https://github.com/dotnet/clicommandlineparser/blob/master/**/*",       
```

Comment:
Why no release branches?

Reply:
We don’t have a release branch moving to VSTS right now.

Reply:
Ok. It'd be fine to add the trigger preemptively, if you want, but this is fine.